rocketc/rocketc.py

```diff
@@ -264,6 +320,8 @@ def init(self):
 
             if user_data["role"] == "instructor" and self.rocket_chat_role != "bot":
                 self.api_rocket_chat.change_user_role(user_id, "bot")
+
```

Comment:
everytime the xblock loads for a user, the grade is calculated? Could it cause performance issues?